notebooks/219-pytorch-object-detection-accuracy-check-and-quantization/219-pytorch-object-detection-accuracy-check-and-quantization.ipynb

```diff
@@ -37,13 +37,26 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "2573d828",
-   "metadata": {
-    "id": "2ynWRum4iiTz"
-   },
+   "id": "1c8cc881",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "![ ! -d 'yolov5' ] && git clone https://github.com/ultralytics/yolov5.git -b v6.1 && cd yolov5 && python export.py --weights yolov5m/yolov5m.pt --imgsz 640 --batch-size 1 --include openvino"
+    "import os\n",
+    "from IPython.display import Markdown, display\n",
+    "\n",
+    "if not os.path.exists(\"./yolov5/\"):\n",
+    "    command_download = f'{\"git clone https://github.com/ultralytics/yolov5.git -b v6.1\"}'\n",
+    "    command_download = \" \".join(command_download.split())\n",
+    "    print(\"Download Ultralytics Yolov5 project source:\")\n",
+    "    display(Markdown(f\"`{command_download}`\"))\n",
+    "    download_res = %sx $command_download\n",
```

Comment:
I would use `!` instead of `%sx` since it gives better console output inside the notebook. (includes ansi colors for the `export.py` script).

Reply:
just modify "download_res = %sx $command_download" to "%sx $command_download" can print in console. Please check with new updated.

Reply:
I don't believe that this is better. This is the output I'm getting (notice the raw ANSI escape codes):

```
["\x1b[34m\x1b[1mexport: \x1b[0mdata=data/coco128.yaml, weights=['yolov5m/yolov5m.pt'], imgsz=[640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['openvino']",
 'YOLOv5 🚀 v6.1-0-g3752807 torch 1.7.1+cpu CPU',
 '',
 'Fusing layers... ',
 'Model Summary: 290 layers, 21172173 parameters, 0 gradients',
 '',
 '\x1b[34m\x1b[1mPyTorch:\x1b[0m starting from yolov5m/yolov5m.pt with output shape (1, 25200, 85) (42.8 MB)',
 '',
 '\x1b[34m\x1b[1mONNX:\x1b[0m starting export with onnx 1.9.0...',
 '\x1b[34m\x1b[1mONNX:\x1b[0m export success, saved as yolov5m/yolov5m.onnx (85.1 MB)',
 '',
 '\x1b[34m\x1b[1mOpenVINO:\x1b[0m starting export with openvino 2022.1.0-7019-cdb9bec7210-releases/2022/1...',
 '\x1b[34m\x1b[1mOpenVINO:\x1b[0m export success, saved as yolov5m/yolov5m_openvino_model/ (85.5 MB)',
 '',
 'Export complete (3.98s)',
 'Results saved to \x1b[1m/home/ncgalopp/code/openvino_notebooks/notebooks/219-pytorch-object-detection-accuracy-check-and-quantization/yolov5/yolov5m\x1b[0m',
 'Detect:          python detect.py --weights yolov5m/yolov5m_openvino_model/',
 "PyTorch Hub:     model = torch.hub.load('ultralytics/yolov5', 'custom', 'yolov5m/yolov5m_openvino_model/')",
 'Validate:        python val.py --weights yolov5m/yolov5m_openvino_model/',
 'Visualize:       [https://netron.app']](https://netron.app%27]/)
```